source/Textures.js

```diff
@@ -0,0 +1,20 @@
+var Pixi = require("pixi.js")
+
+import Junkership from "./Junkership.js"
+
+exports.initTex = function initTex() {
+    PIXI.loader
+        .add("redJunkership",    require("./images/red-starship.png"))
+        .add("yellowJunkership", require("./images/yellow-starship.png"))
+        .add("greenJunkership",  require("./images/green-starship.png"))
+        .add("blueJunkership",   require("./images/blue-starship.png"))
+        .add("projectile",       require("./images/projectile.png"))
+        .add("trashbot",         require("./images/enemy-starship.png"))
+        .load()
+
+        .on("complete",setup)
+}
+
+function setup() {
+    game.spawnWave()
```

Comment:
We could also put generate Junkership here (or allow players to activate Junkerships here) so that there is never a case the Junkership is instantiated before their texture is loaded (and thus avoid the `if (this.width === 1)` kludge.)